lib/modules/platform/azure/index.ts

```diff
@@ -745,7 +745,8 @@ export function massageMarkdown(input: string): string {
       'rename PR to start with "rebase!"'
     )
     .replace(regEx(`\n---\n\n.*?<!-- rebase-check -->.*?\n`), '')
-    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '');
+    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '')
+    .replace(regEx(/<!--renovate:start-->\s*|\s*<!--renovate:end-->/g), '');
```

Comment:
```suggestion
    .replace(regEx(/<!--renovate:start-->.+<!--renovate:end-->/g), '');
```
isn't this better?

Reply:
The goal of this function is to remove unsupported tags. This is what I understood. e.g., 

```
<!--renovate:start-->
Renovate Body
<!--renovate:end-->
```

to 

```
Renovate Body
```

without the `\s*` it would have new lines which are not needed. e.g.,

```

Renovate Body

```

The regex you are suggesting woudl remove everyting:

```
```

We can discuss the need of `\s*` but I think it makes sense to not add unnecessary new lines.